aten/src/ATen/native/SharedReduceOps.h

```diff
@@ -392,8 +392,8 @@ struct NormTwoOps {
 
 template <typename acc_t, typename data_t>
 struct NanSumOps {
-  inline C10_DEVICE acc_t reduce(acc_t a, data_t b, int64_t /*idx*/) const {
```

Comment:
this is actually fine, `reduce` is called only to add a single number `b` which would be representable in `data_t` to accumulator